synapse/models/entity.py

```diff
@@ -23,21 +23,25 @@
 
                     ('name', ('meta:name', {}), {
                         'alts': ('names',),
-                        'doc': 'The primary name of the {contactable}.'}),
+                        'doc': 'The primary entity name of the {contactable}.'}),
 
                     ('names', ('array', {'type': 'meta:name', 'uniq': True, 'sorted': True}), {
-                        'doc': 'An array of alternate names for the {contactable}.'}),
+                        'doc': 'An array of alternate entity names for the {contactable}.'}),
 
                     ('title', ('entity:title', {}), {
-                        'doc': 'The title or role for this {contactable}.'}),
+                        'doc': 'The entity title or role for this {contactable}.'}),
 
                     ('titles', ('array', {'type': 'entity:title', 'uniq': True, 'sorted': True}), {
-                        'doc': 'An array of alternate job titles or roles for this {contactable}.'}),
+                        'doc': 'An array of alternate entity titles or roles for this {contactable}.'}),
 
                     ('url', ('inet:url', {}), {
                         'doc': 'The primary url for the {contactable}.'}),
 
                     ('lifespan', ('ival', {}), {
+                        'virts': {
+                            'min': (None, {'doc': 'The date of birth for an individual or founded date for an organization.'}),
+                            'max': (None, {'doc': 'The date of death for an individual or disolved date for an organization.'}),
```

Comment:
define a doc for duration here as well?

Should these be overridden on the inheritors to be more specific too?

Reply:
I was also debating pumbing a templating system for these as well, but i think it can wait for now. In general the "parent" doc should at lease make sense.